scripts/sct_fmri_moco.py

```diff
@@ -12,81 +12,109 @@
-                      mandatory=False,
-                      default_value=param_default.group_size)
-    parser.add_option(name='-m',
-                      type_value='image_nifti',
-                      description='Binary mask to limit voxels considered by the registration metric.',
-                      mandatory=False)
-    parser.add_option(name='-param',
-                      type_value=[[','], 'str'],
-                      description="Advanced parameters. Assign value with \"=\"; Separate arguments with \",\"\n"
-                                  "poly [int]: Degree of polynomial function used for regularization along Z. For no regularization set to 0. Default=" + param_default.poly + ".\n"
-                                  "smooth [mm]: Smoothing kernel. Default=" + param_default.smooth + ".\n"
-                                  "iter [int]: Number of iterations. Default=" + param_default.iter + ".\n"
-                                  "metric {MI, MeanSquares, CC}: Metric used for registration. Default=" + param_default.metric + ".\n"
-                                  "gradStep [float]: Searching step used by registration algorithm. The higher the more deformation allowed. Default=" + param_default.gradStep + ".\n"
-                                  "sampling [None or 0-1]: Sampling rate used for registration metric. Default=" + param_default.sampling + ".\n"
-                                  "numTarget [int]: Target volume or group (starting with 0). Default=" + param_default.num_target + ".\n"
-                                  "iterAvg [int]: Iterative averaging: Target volume is a weighted average of the previously-registered volumes. Default=" + str(param_default.iterAvg) + ".\n",
-                      mandatory=False)
-    parser.add_option(name='-ofolder',
-                      type_value='folder_creation',
-                      description='Output path.',
-                      mandatory=False,
-                      default_value='./')
-    parser.add_option(name="-x",
-                      type_value="multiple_choice",
-                      description="""Final interpolation.""",
-                      mandatory=False,
-                      default_value='linear',
-                      example=['nn', 'linear', 'spline'])
-    parser.add_option(name="-r",
-                      type_value="multiple_choice",
-                      description="""Remove temporary files.""",
-                      mandatory=False,
-                      default_value='1',
-                      example=['0', '1'])
-    parser.add_option(name="-v",
-                      type_value="multiple_choice",
-                      description="""Verbose.""",
-                      mandatory=False,
-                      default_value='1',
-                      example=['0', '1', '2'])
+    # parser initialisation
+    parser = argparse.ArgumentParser(
+        description=("Motion correction of fMRI data. Some robust features include:\n"
```

Comment:
Are the brackets necessary here? Also out of curiousity, why switch from `"""` to iterated `"` strings? Did the `SmartFormatter` not do the right thing?

Reply:
I had the same thought! The trouble is, multiline strings capture all whitespace, so to get it to come out properly, it would need to have some funky indentation in-code:

```python
    parser = argparse.ArgumentParser(
        description="""Motion correction of fMRI data. Some robust features include:
    - group-wise (-g)
    - slice-wise regularized along z using polynomial function (-p)
     (For more info about the method, type: isct_antsSliceRegularizedRegistration)
    - masking (-m)
    - iterative averaging of target volume

The outputs of the motion correction process are:
    - the motion-corrected fMRI volumes
    - the time average of the corrected fMRI volumes
    - a time-series with 1 voxel in the XY plane, for the X and Y motion direction (two separate 
    files), as required for FSL analysis.
    - a TSV file with the slice-wise average of the motion correction for XY (one file), that 
    can be used for Quality Control.""",
        formatter_class=SmartFormatter,
        add_help=None,
        prog=os.path.basename(__file__).strip(".py")
    )
```

I tried finding a "best of both worlds" way to use multiline strings, and arrived at this answer when I was digging into this: https://stackoverflow.com/a/2504457 (Hence why I used `(` and `"` -- mostly for readability.) 

There could definitely be a better way out there, though.

Reply:
I don't think the `(` is necessary because it is enclosed by the argument bracket - doesn't really matter. I was under the impression that the `SmartFormatter` did the right thing with triple quoted strings. Otherwise there is `textwrap.dedent`, not that anything is wrong with this style.

Reply:
> I don't think the ( is necessary because it is enclosed by the argument bracket

I didn't realize this was the case, but you're absolutely right. Good to know!

> I don't think it's this PR's job to remove multilabel support from OvR.

I've just checked now, and I couldn't get `SmartFormatter` to respect newlines (with or without `\n`) when using `"""`. Hrm. Well, nbd I think.